src/sqs/deployment.ts

```diff
@@ -16,18 +18,30 @@ export type DeploymentMessagePayload = {
 
 export const deploymentQueueMessageHandler: MessageHandler<DeploymentMessagePayload> = async (context: Context<DeploymentMessagePayload>) => {
 
-	context.log.info("Handling deployment message from the SQS queue")
+
 
 	const messagePayload: DeploymentMessagePayload = context.payload;
 
-	const github = await app.auth(messagePayload.installationId);
+	const {webhookId, jiraHost, installationId} = messagePayload;
+
+	context.log = context.log.child({
+		webhookId,
+		jiraHost,
+		installationId
+	})
```

Comment:
Apologise for the adjacent change here, but that's something I've noticed during the yesterday's incident. 

The idea here is put webhookId and installationId into the context logger, so these parameters will be logged in case of the message failure. (Otherwise we can't filter errors by installation Id)